packages/fx-core/templates/plugins/resource/bot/bicep/bot.bicep

```diff
@@ -0,0 +1,97 @@
+param botServiceName string
+param botServerfarmsName string
+param botWebAppSKU string = 'F1'
+param botServiceSKU string = 'S1'
+param botWebAppName string
+param botAadClientId string
+@secure()
+param botAadClientSecret string
+param botDisplayName string
+param authLoginUriSuffix string
+param m365ClientId string
+@secure()
+param m365ClientSecret string
+param m365TenantId string
+param m365OauthAuthorityHost string
+param m365ApplicationIdUri string
+
+var botWebAppHostname = botWebApp.properties.hostNames[0]
+var botEndpoint = 'https://${botWebAppHostname}'
+var initiateLoginEndpoint = uri(botEndpoint, authLoginUriSuffix)
+
+resource botServices 'Microsoft.BotService/botServices@2021-03-01' = {
+  kind: 'azurebot'
+  location: 'global'
+  name: botServiceName
+  properties: {
+    displayName: botDisplayName
+    endpoint: 'https://${botServiceName}.azurewebsites.net/api/messages'
+    msaAppId: botAadClientId
+  }
+  sku: {
+    name: botServiceSKU
+  }
+}
+
+resource botServices_MsTeamsChannel 'Microsoft.BotService/botServices/channels@2021-03-01' = {
```

Comment:
we should keep same naming style for resources

Reply:
Fixed.